homeassistant/components/blink/coordinator.py

```diff
@@ -0,0 +1,33 @@
+"""Blink Coordinator."""
+from __future__ import annotations
+
+from datetime import timedelta
+import logging
+from typing import Any
+
+from blinkpy.blinkpy import Blink
+
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+
+class BlinkUpdateCoordinator(DataUpdateCoordinator[dict[str, Any]]):
+    """BlinkUpdateCoordinator - In charge of downloading the data for a site, which all the sensors read."""
```

Comment:
Please break long strings around max 88 characters per line.

Reply:
Strange - I thought that was lint/ruff/black's job to clean these up... or at least complain :)